src/fc/qemu/agent.py

```diff
@@ -253,6 +255,20 @@ def tmp_size(disk):
     return int(tmp_gib) * GiB
 
 
+def iproute2_json(*args):
```

Comment:
there is logging infrastructure for subprocess calls around, please check that we leverage this for debugging

Reply:
I'm not a *massive* fan of the way `fc.qemu.util.cmd()` stringifies commands, passes them through the shell, and then reads the result line-wise in text mode for this particular use case, but I've switched to that function instead of calling `subprocess.check_output()` directly so we have the debug logging output.

Reply:
I'd be happy to go through what you don't like about it, maybe we can combine the requirements more cleanly.